html/changelog.html

```diff
@@ -58,6 +58,15 @@
 
 <div class='commit sansserif'>
 	<h2 class='date'>25 July 2015</h2>
+	<h3 class='author'>Nalar updated:</h3>
+	<ul class='changes bgimages16'>
+		<li class='tweak'>Cat-banned people can no longer be antagonists.</li>
+		<li class='tweak'>Cat-banned people are now restricted to assistan only.</li>
```

Comment:
Minor spelling error, missing a T at the end of assistant, but otherwise it looks fine to me.